tests/test_parser.py

```diff
@@ -686,14 +685,7 @@ def test_array_type_spec(self) -> None:
                                 Component(
                                     "Length",
                                     "Byte",
-                                    [
-                                        Then(
-                                            "Bytes",
-                                            UNDEFINED,
-                                            Mul(Variable("Length"), Number(8)),
-                                            BooleanTrue(),
-                                        )
-                                    ],
+                                    [Then("Bytes", UNDEFINED, Mul(Variable("Length"), Number(8)),)],
```

Comment:
I think the trailing comma is not necessary anymore since there is no line break.

Reply:
if i remove it and run `make format`, black puts the comma back in

Reply:
I just tried that: black version 19.10b0 didn't put a comma back there. Usually such an comma is a leftover when black reduces a multiline statement to one line. As far as I remember that is known bug of black.

Reply:
It works now. I tried to remove the wrong trailing comma, sorry :)